remme/ws/__main__.py

```diff
@@ -33,5 +34,7 @@
     app = web.Application(loop=loop)
     stream = Stream(ZMQ_URL)
     ws_handler = WsApplicationHandler(stream, loop=loop)
+    ws_event_handler = WSEventSocketHandler(stream, loop=loop)
+    app.router.add_route('GET', '/ws/events', ws_event_handler.subscriptions)
```

Comment:
Why do not use one route (just `/ws`)?

Reply:
Because it would require adopting a current version of /ws(which was the first attempt), moreover such routing solves the problem of routing requests, rather than adding a separate class for this purpose.